ошибки2

Необходимо исправить ошибку

"
2026-04-07 22:54:09,229 [WARNING] __main__: Could not place circle pair 17 after 1000 attempts. Consider reducing count or increasing drawing area.
2026-04-07 22:54:09,230 [WARNING] __main__: Could not place circle pair 18 after 1000 attempts. Consider reducing count or increasing drawing area.
2026-04-07 22:54:09,232 [WARNING] __main__: Could not place circle pair 19 after 1000 attempts. Consider reducing count or increasing drawing area.
2026-04-07 22:54:09,234 [WARNING] __main__: Could not place circle pair 20 after 1000 attempts. Consider reducing count or increasing drawing area.
2026-04-07 22:54:09,443 [ERROR] __main__: Unexpected error
Traceback (most recent call last):
  File "C:\Users\Администратор\PycharmProjects\kompas_api_random_points\kompas_random_circles.py", line 706, in main
    run_drawing(settings)
  File "C:\Users\Администратор\PycharmProjects\kompas_api_random_points\kompas_random_circles.py", line 445, in run_drawing
    add_new_sheet(
  File "C:\Users\Администратор\PycharmProjects\kompas_api_random_points\kompas_random_circles.py", line 378, in add_new_sheet
    sheet_param = api5_module.ksSheetParam(
                  ^^^^^^^^^^^^^^^^^^^^^^^^
AttributeError: module 'win32com.gen_py.0422828C-F174-495E-AC5D-D31014DBBE87x0x1x0' has no attribute 'ksSheetParam'. Did you mean: 'ksSheetPar'?
Unexpected error: module 'win32com.gen_py.0422828C-F174-495E-AC5D-D31014DBBE87x0x1x0' has no attribute 'ksSheetParam'
"

Насколько я понял новый лист не создаётся, не создаётся новый вид с новым случайным размещением на новом листе

--- a/.gitkeep
+++ b/.gitkeep
@@ -1,1 +1,0 @@
-# .gitkeep file auto-generated at 2026-04-07T20:01:48.069Z for PR creation at branch issue-5-6dfa17078c02 for issue https://github.com/PavelChurkin/kompas_api_random_points/issues/5

